modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-test/src/testIntegration/java/com/liferay/portal/workflow/metrics/rest/resource/v1_0/test/InstanceResourceTest.java

```diff
@@ -397,15 +480,31 @@ protected Instance testGetProcessInstancesPage_addInstance(
 			Long processId, Instance instance)
 		throws Exception {
 
+		return testGetProcessInstancesPage_addInstance(
+			new HashMap<>(), processId, instance);
+	}
+
+	protected Instance testGetProcessInstancesPage_addInstance(
+			Map<Long, List<Long>> assigneeGroupIds, Long processId,
+			Instance instance)
```

Comment:
@seltong, could you put instance parameter before processId?

Reply:
The method below has the same formatting problem, right? Do I have to fix this too?
protected Instance testGetProcessInstancesPage_addInstance(
			Long processId, Instance instance)